src/main/java/org/hydev/mcpm/client/arguments/fetcher/BriefFetcherListener.java

```diff
@@ -1,4 +1,4 @@
-package org.hydev.mcpm.client.database.fetcher;
+package org.hydev.mcpm.client.arguments.fetcher;
 
 /**
  * Stub implementation of DatabaseFetcherListener.
```

Comment:
why were these classes in `client.database.fetcher` moved to `client.arguments.fetcher`? What do they have to do with arguments?

Reply:
oh my bad. i must have moved the directory by accident. this definitely shouldn't be